src/MongoDB.Driver.Legacy/MongoServerSettings.cs

```diff
@@ -403,6 +405,19 @@ public string ReplicaSetName
             }
         }
 
+        /// <summary>
+        /// Gets or sets whether to retry reads.
+        /// </summary>
+        public bool RetryReads
```

Comment:
No `<value/>`

Reply:
Yep. I just not sure when we should add this part and when not.
Anyway, I agree with your solution.